packages/docs/examples/AddAsset.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
+import { cssObj } from '@fuel-ui/css';
+import { Box, Stack, Button, Input, Tag } from '@fuel-ui/react';
+import { useState } from 'react';
+
+import type { Asset } from '~/../types/src';
+import { ExampleBox } from '~/src/components/ExampleBox';
+import { useFuel } from '~/src/hooks/useFuel';
+import { useIsConnected } from '~/src/hooks/useIsConnected';
+import { useLoading } from '~/src/hooks/useLoading';
+
+export function AddAsset() {
+  const [fuel, notDetected] = useFuel();
+  const [isConnected] = useIsConnected();
+  const [addedAsset, setAddedAsset] = useState<Asset>();
+  const [asset, setAsset] = useState<Asset>({
+    name: 'New',
+    symbol: 'NEW',
+    assetId:
+      '0x566012155ae253353c7df01f36c8f6249c94131a69a3484bdb0234e3822b5d90',
+    imageUrl:
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/4/46/Bitcoin.svg/1200px-Bitcoin.svg.png',
```

Comment:
Can we add this asset config to some mock file and reuse it in all places?

Reply:
added mock in `playwright/e2e` and `src/app/assets/mock`